.buildkite/test-static-sanitized.sh

```diff
@@ -12,10 +12,10 @@ esac
 
 if [[ "linux" == "$platform" ]]; then
   CONFIG_OPTS="--config=buildfarm-sanitized-linux"
-  RUBY_TESTS=""
+  MAC_TESTS=""
```

Comment:
We've recently changed mac tests to only run in master.
This isn't going to be tested in branches.

How long do you expect it to be only tested on mac? should we re-enable branch tests on mac?

Reply:
We could re-enable the branch tests on mac for now, moving these tests to linux will just require fixing that issue with `probes.h` on the ruby build.

Reply:
Happy to re-enable them for now. Please edit `pipeline.yaml` for that removing branch filter

Reply:
I've pushed a change to re-enable it for branches. Once @ruby_2_4_3 works on linux, we can move it back to `master` only.